apps/metrics/__init__.py

```diff
@@ -0,0 +1,107 @@
+class MetricType:
+    UNKNOWN = 'unknown'
+    TIME = 'time'
+    SIZE = 'size'
+    COUNT = 'count'
+
+
+class MetricCollection(object):
```

Comment:
We could derive from UserList (https://docs.python.org/3.1/library/collections.html#collections.UserList) to have more list-like interface.